Sources/Mantis/CropViewConfig.swift

```diff
@@ -0,0 +1,20 @@
+import UIKit
+
+public struct CropViewConfig {
+    public var cropViewMinimumBoxSize: CGFloat = 42
+    public var cropViewPadding: CGFloat = 14
```

Comment:
Maybe? Since it's now part of `CropViewConfig`

```suggestion
    public var padding: CGFloat = 14
```